page_analyzer/app.py

```diff
@@ -1,8 +1,76 @@
-from flask import Flask, render_template
+import os
+from typing import Any, Dict
+
+from dotenv import load_dotenv
+from flask import Flask, flash, redirect, render_template, request, url_for
+from page_analyzer.utils.db import connect_db
+from page_analyzer.utils.url import normalize, validate
+
+load_dotenv()
+
 
 app = Flask(__name__)
+app.secret_key = os.getenv("SECRET_KEY")
+
+
+def get_url_context(url_id: int) -> Dict[str, str]:
+    conn = connect_db()
+    cur = conn.cursor()
+    cur.execute("SELECT name, created_at FROM urls WHERE id='%s';", (url_id,))
```

Comment:
[just an idea] please, take a look at the [named parameters section](https://www.psycopg.org/docs/usage.html#passing-parameters-to-sql-queries) and decide what you like more - unnamed or named parameters